File: 03_Spring/Backend_Course/assignment1/src/main/java/com/example/assignment1/board/BoardService.java
package com.example.assignment1.board;

import java.util.List;

public interface BoardService {
    public void insertBoard(BoardVO vo);
    public void deleteBoard(BoardVO vo);
    public void updateBoard(BoardVO vo);
    public BoardVO getBoard(BoardVO vo);
    public List<BoardVO> getBoardContentList(BoardVO vo);
    public List<BoardVO> getBoardList(BoardVO vo);
}

File: 03_Spring/Backend_Course/assignment1/src/main/java/com/example/assignment1/board/BoardService.kt
package com.example.assignment1.board

interface BoardService {
    fun insertBoard(vo: BoardVO)
    fun deleteBoard(vo: BoardVO)
    fun updateBoard(vo: BoardVO)
    fun getBoard(vo: BoardVO): BoardVO?
    fun getBoardContentList(vo: BoardVO): List<BoardVO>?
    fun getBoardList(vo: BoardVO?): List<BoardVO>?
}